app/src/main/res/values-si/string.xml

```diff
@@ -0,0 +1,1027 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="app_name">PSLab</string>
+    <string name="openDrawer">open_drawer</string>
+    <string name="closeDrawer">close_drawer</string>
+
+    <string name="Toast_double_tap">PSLab වසා දැමීමට නැවත ඔබන්න</string>
+    <string name="Toast_magnetic_not_present">"මෙම උපකරණය සඳහා අවශ්‍ය සංවේදකය ඔබගේ උපාංගයේ නොමැත"</string>
+
+    <string name="nav_device">උපාංගය සම්බන්ධ කරන්න</string>
+    <string name="nav_instruments">උපකරණ</string>
+    <string name="nav_settings">සැකසුම්</string>
+    <string name="nav_about_us">අප ගැන</string>
+    <string name="nav_buy_pslab">PSLab මිලදී ගන්න</string>
+    <string name="nav_report">දෝෂ හා ප්‍රතිචාර</string>
+    <string name="nav_help_feedback">නිතර අසන ලද ප්‍රශ්න</string>
+    <string name="nav_share">යෙදුම බෙදාගන්න</string>
+    <string name="nav_config">වින්යාස ගොනුව උත්පාදනය කරන්න</string>
+    <string name="pslab_pinlayout">පිරිසැලසුම පින් කරන්න</string>
+    <string name="pslab_pinlayout_front">පිරිසැලසුම ඉදිරියෙන් පින් කරන්න</string>
+    <string name="pslab_pinlayout_back">පිරිසැලසුම පසුපසින් පින් කරන්න</string>
+    <string name="bluetooth_connect_menu">බ්ලූටූත් සම්බන්ධතාවය</string>
+    <string name="bluetooth_wifi_connect_option">බ්ලූටූත් හෝ වයි-ෆයි භාවිතයෙන් සම්බන්ධ වන්න</string>
+    <string name="bluetooth">බ්ලූටූත්</string>
+    <string name="wifi">වයි-ෆයි</string>
+    <string name="bluetooth_scan_text">සුපරීක්ෂණය කරන්න</string>
+    <string name="bluetooth_stop_text">නවත්වන්න</string>
+    <string name="scanned_devices_list_title">සුපරීන්ෂණය කළ උපාංග</string>
+    <string name="bluetooth_not_supported">ඔබගේ දුරකථනය බ්ලූටූත් සඳහා සහය නොදක්වයි</string>
+    <string name="incorrect_IP_address_message">කරුණාකර වලංගු IP ලිපිනය ඇතුළත් කරන්න</string>
+    <string name="enter_IP_hint">ඊඑස්පී මොඩියුලයේ IP ලිපිනය ඇතුළත් කරන්න</string>
+    <string name="esp_connect_text">සම්බන්ධ කරන්න</string>
+
+    <string name="oscilloscope">ඔසිලෝස්කෝප්</string>
```

Comment:
Can we use 'දෝලනේක්ෂය' instead of 'ඔසිලෝස්කොප්'?

Reply:
yeah seems good